code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -299,78 +299,73 @@
 	heal_burn = 25
 	heal_oxy = 25
 
-//Warp Whistle: Provides uncontrolled long distance teleportation.
-/obj/item/warpwhistle
+///Warp whistle, spawns a tornado that teleports you
+/obj/item/warp_whistle
 	name = "warp whistle"
-	desc = "One toot on this whistle will send you to a far away land!"
```

Comment:
:-1: you also removed the reference description

Reply:
I preferred being descriptive rather than just shitposting.